invenio.cfg

```diff
@@ -182,6 +184,7 @@ SECURITY_RECOVERABLE = False  # local login: allow users to reset the password
 SECURITY_CHANGEABLE = False  # local login: allow users to change psw
 SECURITY_CONFIRMABLE = False  # local login: users can confirm e-mail address
 SECURITY_LOGIN_WITHOUT_CONFIRMATION = True # require users to confirm email before being able to login
+ACCOUNTS_DEFAULT_USER_VISIBILITY = "public" # enables users to be searchable for invites
```

Comment:
This will enable users to allow to search for them by name, however there is another variable to allow to search by email (and in consequence exposes the email) do we want to allow this as well?

In another note, at the moment in CDS-RDM we are not confirming the users when syncing or logging in, there is actually no way to confirm the account. Without confirming the acount the users are not searchable, if I am not mistaken it's part of the logic to make it searchable. Should we automatically confirm the users internally (when logging in and syncing)? Should we enable the normal confirmation (if we do this, synced users won't be searchable as they have to first confirm the accounts)? 

Alternatively we could change the logic to not require user confirmation to make it searchable. 

ping @kpsherva

Reply:
to be verified that public visibility means only visible for logged in users (not 100% public)

Reply:
If not logged in we get a 403 when fetching the users API (api/users)